### PR TITLE
Issue #13213: Remove '//ok' comments from InputRequireThisTryWithResources

### PR DESCRIPTION
**Pull request by HRuii1**
Issue #13213

### CODE REVIEW

File: config/checkstyle-input-suppressions.xml [hunk 1]
```diff
@@ -164,52 +164,6 @@
             files="checks[\\/]coding[\\/]requirethis[\\/]InputRequireThisReceiver.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]coding[\\/]requirethis[\\/]InputRequireThisStatic.java"/>
-  <suppress id="UnnecessaryOkComment"
```
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:167]: Everything lgtm....

**Review by MANISH-K-07 — COMMENTED** [review]
The CI are all successful.

All it needs is an approval from maintainers

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by HRuii1** [comment]
conflict resolved

**Comment by romani** [comment]
Nope :(.

Do it by rebase https://checkstyle.org/beginning_development.html#Starting_Development

**Comment by HRuii1** [comment]
should be good now

**Comment by romani** [comment]
CI is very red please investigate and resolve

**Comment by romani** [comment]
@HRuii1, please finish PR

**Comment by romani** [comment]
Some other PR did this update

**Pull request closed without merging**